tests/pgautofailover_utils.py

```diff
@@ -70,6 +71,30 @@ def create_datanode(self, datadir, port=5432, group=0,
         self.datanodes.append(datanode)
         return datanode
 
+    def pg_createcluster(self, datadir, port=5432):
+        """
+        Initializes a postgresql node using pg_createcluster and returns
+        directory path.
+        """
+        vnode = self.vlan.create_node()
+        nodeid = len(self.datanodes) + 1
+
+        create_command = ["sudo", shutil.which('pg_createcluster'),
+                          "-U", os.getenv("USER"),
```

Comment:
you mentioned an error earlier `Error: clusters must not be owned by root` this line is the reason for that.

Perhaps we should `postgres` user if  current user is root.

Reply:
The problem was with using "docker" hard-coded there, where we need to use `os.getenv("USER")` like in the other call sites.